lib/klaviyo/apis/identify.rb

```diff
@@ -0,0 +1,26 @@
+class Identify < Klaviyo::Client
+  # Used for identifying customers and managing profile properties
+  #
+  # @kwarg :id [String] the customer or profile id
+  # @kwarg :email [String] the customer or profile email
+  # @kwarg :properties [Hash] properties of the profile to add or update
+  def self.identify(kwargs = {})
+    defaults = {:id => nil, :email => nil, :properties => {}}
+    kwargs = defaults.merge(kwargs)
+
+    if kwargs[:email].to_s.empty? and kwargs[:id].to_s.empty?
+      raise KlaviyoError.new('You must identify a user by email or ID')
+    end
+
+    properties = kwargs[:properties]
+    properties[:email] = kwargs[:email] unless kwargs[:email].to_s.empty?
+    properties[:id] = kwargs[:id] unless kwargs[:id].to_s.empty?
```

Comment:
Similar logic here as there is in the Track class. Some helper methods could de-duplicate this logic.